opdracht-2.js

```diff
@@ -0,0 +1,64 @@
+// Opdracht 2.A ------------------------------------------
+// 2.A.1 - Maak een array met verkochte tv's per model:
+const soldPerModel = inventory.map((tv) => {
+    return tv.sold;
+});
```

Comment:
Nog een kleine tip: als je in een function-body maar één regel hebt, dan kun je het ook op deze manier doen
```suggestion
const soldPerModel = inventory.map(tv => tv.sold);
```